server/src/main/java/org/opensearch/rest/action/admin/cluster/RestDecommissionAction.java

```diff
@@ -58,6 +58,7 @@ DecommissionRequest createRequest(RestRequest request) throws IOException {
             TimeValue delayTimeout = request.paramAsTime("delay_timeout", DecommissionRequest.DEFAULT_NODE_DRAINING_TIMEOUT);
             decommissionRequest.setDelayTimeout(delayTimeout);
         }
-        return decommissionRequest.setDecommissionAttribute(new DecommissionAttribute(attributeName, attributeValue));
+        return decommissionRequest.setDecommissionAttribute(new DecommissionAttribute(attributeName, attributeValue))
+            .originalRequest(false);
```

Comment:
nit : no need to ` .originalRequest(false);` as it is already the default value .

Reply:
Yes. Removing